class ValueNotFound(Exception):
    pass

class BadResponse(Exception):
    pass